portfolio/src/main/java/com/google/sps/usercomment/UserComments.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.usercomment;
+
+/**
+* Class object to define the structure of a user comment.
+*/
+public class UserComments {
+  public String user;
+  public String comment;
+  public long id;
+
```

Comment:
Consider make private these three variables

Reply:
I got error when I tried that. In this case the solution will be to make the class as private, which seems unnecessary.